### PR TITLE
Adding command to copy the current class name to clipboard

### LINKED ISSUE #922: [vim] Copy FQN to clipboard command
Not sure how often this would be useful.

### PR DESCRIPTION
**Pull request by mamazu**
Fixes #922

First try at vim script, so please tell me if I made some mistakes.

This command assumes that the clipboard register is the `"+` register. Maybe we should make that configurable.

### CODE REVIEW

File: ftplugin/php/commands.vim [hunk 1]
```diff
@@ -65,3 +65,7 @@ command! -buffer -nargs=0 PhpactorGenerateAccessors call phpactor#GenerateAccess
 ""
 " Automatically add any missing properties to a class
 command! -buffer -nargs=0 PhpactorTransform call phpactor#Transform()
```
Comment (dantleech) [inline ftplugin/php/commands.vim:67]: This text is used tl generate the docs, can be changed to : Copy the current class FQN (based on current filename)  to the clipboard

File: ftplugin/php/commands.vim [hunk 2]
```diff
@@ -65,3 +65,7 @@ command! -buffer -nargs=0 PhpactorGenerateAccessors call phpactor#GenerateAccess
...
+
+""
+" Getting the full name of the class
```
Comment (dantleech) [inline ftplugin/php/commands.vim:70]: This text is used tl generate the docs, can be changed to : Copy the current class FQN (based on current filename) to the clipboard

**Review by dantleech — CHANGES_REQUESTED** [review]
Looks good to me! Need to run "make docs"  (or make vimdoc) to regenerate the vim documentation, which needs to be committed.

**Comment by mamazu** [comment]
I also moved the command up to sort them alphabetically.

**Comment by dantleech** [comment]
Nice, thanks. Test is failure is an unrelated, known, flakey test.

**Pull request merged**